src/App.tsx

```diff
@@ -23,7 +32,7 @@ const EventPage = () => {
     <MainLayout>
```

Comment:
The `EventPage` component is not used anymore. I would suggest to simply remove it.